Composer/packages/client/src/pages/design/DebugPanel/TabExtensions/WebChatLog/WebChatLogContent.tsx

```diff
@@ -226,8 +233,18 @@ export const WebChatLogContent: React.FC<DebugPanelTabHeaderProps> = ({ isActive
         </div>
       );
     }
+
+    if (isBotStarting(currentStatus)) {
+      return (
+        <div css={emptyStateMessageContainer}>
+          {formatMessage('{total, plural, =1 {Starting your bot...}other {Starting your bots...}}', {
+            total: runningBots.totalBots,
+          })}
+        </div>
+      );
+    }
     return null;
-  }, [currentStatus]);
+  }, [currentStatus, currentProjectId]);
```

Comment:
should this be aware of runningBots changes?

Reply:
![Screen Shot 2021-06-21 at 2 51 04 PM](https://user-images.githubusercontent.com/13004779/122832419-23134d80-d2a0-11eb-8c1d-d72e4c8e17ed.png)
It has to since the messaging here changes based on the bot status